ledger-tool/src/bigtable.rs

```diff
@@ -263,6 +273,31 @@ impl BigTableSubCommand for App<'_, '_> {
                                 ),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("delete")
```

Comment:
```suggestion
                    SubCommand::with_name("delete-slots")
```
Let's not claim the whole namespace